docs/_includes/head.html

```diff
@@ -69,7 +59,12 @@
       };
       anchors.add("h1, h2, h3, h4, h5");
     });
```

Comment:
Lines 20 through 61 are formatting changes that ideally are excluded. We're in need of enforcement using a common prettier precommit hook, @nebula-aac